symphony/data/datasets/cath.py

```diff
@@ -152,20 +152,59 @@ def parse_pdb_format(line):
     atom_types = ["N", "CA", "C", "CB"]
     amino_acid_abbr = CATHDataset.get_amino_acids()
     all_structures = []
+
+    def _add_structure(pos, spec, molfile):
```

Comment:
stylistically, I don't like underscores at the beginning of functions internal to another function (since they cannot be imported anyways): _add_structure -> add_structure

Reply:
but more of a nitpick